File: Heavyweights/rhinoceros/rhinoceros.py

class Rhinoceros:
    def __init__(self):
        self.form = "Metallic Beast"
        self.speed_in_jungle = "High"
        self.speed_in_lanes = "Reduced"
        self.stamina_generator = True

    def breakdown(self, collided_with_wall):
        if collided_with_wall:
            return "Damage and Stun"
        else:
            return "Damage and Armor Break"

    def to_raise(self):
        return "Provokes enemies and disables their attacks"

    def rear_stupidity(self, near_wall):
        if near_wall:
            return "Damage, Slow and Disorientation"
        else:
            return "Damage only"

    def ultimate(self):
        return "Splits into 3, charges long distance, deals Damage, Slow, Armor and Magic Resist reduction"

File: Heavyweights/rhinoceros/Rhinoceros.java

public class Rhinoceros {
    private String form = "Metallic Beast";
    private String speedInJungle = "High";
    private String speedInLanes = "Reduced";
    private boolean staminaGenerator = true;

    public String breakdown(boolean collidedWithWall) {
        if (collidedWithWall) {
            return "Damage and Stun";
        } else {
            return "Damage and Armor Break";
        }
    }

    public String toRaise() {
        return "Provokes enemies and disables their attacks";
    }

    public String rearStupidity(boolean nearWall) {
        if (nearWall) {
            return "Damage, Slow and Disorientation";
        } else {
            return "Damage only";
        }
    }

    public String ultimate() {
        return "Splits into 3, charges long distance, deals Damage, Slow, Armor and Magic Resist reduction";
    }
}
